server/routes/testutils/appSetup.ts

```diff
@@ -53,6 +56,16 @@ class MockUserService extends UserService {
   }
 }
 
+class MockSupportedPrisonsService extends SupportedPrisonsService {
```

Comment:
Using this mock service here only for passing to the `standardRouter` so that the `populatedSelectedEstablishment` middleware can work for `supertest` tests on routes.

Will have to be revisited when looking at refactoring along the lines of https://github.com/ministryofjustice/hmpps-template-typescript/pull/106.